website/project/model.py

```diff
@@ -3169,6 +3229,11 @@ def add_authorizer(self, user, approved=False, save=False):
         return False
 
     def remove_authorizer(self, user):
+        """Remove a user as an authorizer
```

Comment:
Is there a rationale for keeping the unused code? I mean, if it's *totally* unused....